docs/source/contrib_docs/getting-started.rst

```diff
@@ -1,22 +1,14 @@
-=================================
```

Comment:
Likewise, it's not a good idea to have 'new' in the URL (like `developer-docs-new`), because some day that part won't be new.